lib/manager/bundler/extract.ts

```diff
@@ -46,17 +45,17 @@ export async function extractPackageFile(
       const dep: PackageDependency = {
         depName: gemMatch.groups.depName,
         managerData: { lineNumber },
+        datasource: RubyGemsDatasource.id,
```

Comment:
Why this is now required?

Reply:
Before this change it was just added if `dep.skipReason` was not set. But with this change `skipReason` is never set, meaning that `datasource` will be always added. So I refactored it a bit to avoid the unnecessary condition below.

Or did I missed something here?